commit 95a00cecd9f0668107d11ee68dfed50a525f8bde (refs/changes/21/4221/2)
Author: dyep49 <dyep49@gmail.com>
Date:   2018-06-15T12:59:44-07:00 (1 year, 4 months ago)
    
    TRITON-491 Add metricPorts metadata to VMAPI for cmon-agent
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/boot/setup.sh b/boot/setup.sh
index c960151..39d8ba4 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -7,7 +7,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
@@ -42,6 +42,9 @@ sdc_log_rotation_add registrar /var/svc/log/*registrar*.log 1g
 sdc_log_rotation_add $role /var/svc/log/*$role*.log 1g
 sdc_log_rotation_setup_end
 
+# Add metadata for cmon-agent discovery
+mdata-put metricPorts 8881
+
 # All done, run boilerplate end-of-setup
 sdc_setup_complete
 
diff --git a/package.json b/package.json
index 9042f19..c17bdd7 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.5.2",
+  "version": "9.5.3",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -27,7 +27,7 @@
     "sprintf": "0.1.1",
     "strsplit": "1.0.0",
     "trace-event": "1.3.0",
-    "triton-metrics": "0.1.0",
+    "triton-metrics": "0.1.1",
     "triton-tags": "^1.1.4",
     "vasync": "2.2.0",
     "verror": "1.10.0",
